Common/anaTupleProducer.py

```diff
@@ -186,6 +187,16 @@ def createAnatuple(inFile, outFile, config, sample_name, snapshotOptions,range,
     isData = False
     with open(args.configFile, 'r') as f:
         config = yaml.safe_load(f)
+    config_global = config['GLOBAL']
+    customisations= args.customisations.split(',')
```

Comment:
`ApplyConfigCustomisations` is a general function, which we can use in different places. Put it in utilities
```
def ApplyConfigCustomisation(config_dict, customisations):
    ...
```

Reply:
ok